core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -3773,6 +3774,35 @@ public void checkOrderBy(final boolean desc,
             "empid=110; name=Theodore");
   }
 
+  @Test void testExceptToDistinct() {
```

Comment:
Isn't this test covering `MinusToDistinctRule`? If so, the test name should be `testMinusToDistinct` or similar. Please check.

Reply:
I agree with @asolimando . Although the meanings of  `MINUS` and `Except` are the same, We still need to stay consistent.

Reply:
> testMinusToDistinct

It makes sense to me,I have changed name to testMinusToDistinct